### PR TITLE
added 0.56 release notes

### PR DESCRIPTION
**Pull request by hellt**

### CODE REVIEW

File: docs/manual/vrnetlab.md [hunk 1]
```diff
@@ -29,29 +29,30 @@ Containerlab depends on `hellt/vrnetlab` project, and sometimes features added i
...
+| `0.53.0`         | [`0.15.0`](https://github.com/hellt/vrnetlab/releases/tag/v0.15.0) | Added support for [Fortigate](kinds/fortinet_fortigate.md), [freebsd](kinds/freebsd.md), added lots of FP5 types to Nokia SR OS and support for external cf1/2 disks                   |
+| `0.54.0`         | [`0.16.0`](https://github.com/hellt/vrnetlab/releases/tag/v0.16.0) | Added support for [Cisco c8000v](kinds/c8000.md)                                                                                                                                       |
+| `0.55.0`         | [`0.17.0`](https://github.com/hellt/vrnetlab/releases/tag/v0.17.0) | Added support for [Juniper vJunos-router](kinds/vr-vjunosrouter.md), [Generic VM](kinds/generic_vm.md), support for setting qemu parameters via env vars nodes                         |
+| `0.56.0`         | [`0.18.0`](https://github.com/hellt/vrnetlab/releases/tag/v0.18.0) | Added support for [Dell SONiC](kinds/dell_sonic.md), [SONiC VM](kinds/sonic-vm.md), [Cisco Catalyst 9000v](kinds/vr-cat9kv.md), support for setting qemu parameters via env vars nodes |
```
Comment (vista-) [inline docs/manual/vrnetlab.md:55]: I think the qemu envvar change is not applicable for this release

File: docs/rn/0.56.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Release 0.56
+
+:material-calendar: 2024-07-09 · :material-list-status: [Full Changelog](https://github.com/srl-labs/containerlab/releases)
+
+## Interface aliases
+
+We know, it has been a long time coming, but thatnks to a brilliant contribution done by [@vista-](https://github.com/vista-) in #2124 we now have added support for [interface aliases](../manual/topo-def-file.md#interface-naming).
```
Comment (vista-) [inline docs/rn/0.56.md:7]: *thatnks -> thanks :)

File: docs/rn/0.56.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+--8<-- "docs/manual/topo-def-file.md:aliases"
+
+Interface aliases will be also displayed in the [generated graph](../cmd/graph.md#html) and even in the [edgeshark](../manual/wireshark.md#edgeshark-integration) web view.
+
+/// tab | Graph
+![pic1](https://gitlab.com/rdodin/pics/-/wikis/uploads/9ce9f8bd9ff7e4adfbe45c46712016b0/image.png){.img-shadow}
+///
+/// tab | Edgeshark
+![pic2](https://gitlab.com/rdodin/pics/-/wikis/uploads/3d4541b11a1b557967e419165aeb7ffd/image__2___1___1_.png){.img-shadow style=height:500px}
```
Comment (vista-) [inline docs/rn/0.56.md:19]: oh wow, I didn't even realise this was a thing :)

File: docs/rn/0.56.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+///
+
+## Ostinato support
+
+We have added support for [Ostinato](https://ostinato.org/) traffic generator in containerlab. You can now define Ostinato nodes in your topology file and use them to generate traffic between your nodes.
+
+See the [Ostinato integration](../manual/kinds/ostinato.md) page for more details.
+
+## Sonic
```
Comment (vista-) [inline docs/rn/0.56.md:28]: *SONiC

File: docs/rn/0.56.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+We have added support for the following new Sonic kinds:
```
Comment (vista-) [inline docs/rn/0.56.md:30]: SONiC

File: docs/manual/vrnetlab.md (hunk 1 above)
Comment (hellt) [reply docs/manual/vrnetlab.md:55]: tru that, copy pasting too fast

File: docs/rn/0.56.md (hunk 3 above)
Comment (hellt) [reply docs/rn/0.56.md:19]: yeah, edgeshark is awesome

File: docs/rn/0.56.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Release 0.56
+
+:material-calendar: 2024-07-09 · :material-list-status: [Full Changelog](https://github.com/srl-labs/containerlab/releases)
+
+## Interface aliases
+
+We know, it has been a long time coming, but thanks to a brilliant contribution done by [@vista-](https://github.com/vista-) in #2124 we now have added support for [interface aliases](../manual/topo-def-file.md#interface-naming).
+
+What are interface aliases you ask? They are a way to use interface names as you see them in the NOS CLI, rather than the default `ethX` naming scheme. It is a convenience feature, but, oh boy, how much easier it is to define links when you don't have to keep doing this mental gymnastics of mapping `eth0` to `Gi0/1` and so on.
```
Comment (kaelemc) [inline docs/rn/0.56.md:9]: Maybe how much easier _is it_ versus _it is_?
and 'this mental gymnastics' to 'the mental gymnastics'

File: docs/manual/kinds/vr-cat9kv.md
Comment (kaelemc) [inline docs/manual/kinds/vr-cat9kv.md:1]: 'vrnetlab/cat9kv' should probably link to [the readme file](https://github.com/hellt/vrnetlab/blob/master/cat9kv/README.md)

Comment (kaelemc) [reply docs/manual/kinds/vr-cat9kv.md:1]: Under ASIC simulation config, didn't highlight the lines for some reason.

Comment (hellt) [reply docs/manual/kinds/vr-cat9kv.md:1]: which part exactly @kaelemc ?

File: docs/rn/0.56.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- [`sonic-vm`](../manual/kinds/sonic-vm.md) - is a VM-based upstream SONiC. You may want to pick sonic-vm over [sonic-vs](../manual/kinds/sonic-vs.md) if you hit some issues with the original sonic container #2120
+- [`dell_sonic`](../manual/kinds/dell_sonic.md) - is a SONiC distribution by Dell #2125
+
+## Multi-arch `clab` container build
+
+To support the imminent release of ARM64 friendly Network OSes we fine tuned the release pipeline to make sure the `clab` container is built for both `amd64` and `arm64` architectures. #2128
+
+## Cisco Catalyst 9000v
+
+Thanks to our strong community and @kaelemc in particular, we have added support for the [Cisco Catalyst 9000v](../manual/kinds/vr-cat9kv.md) router. #2133
```
Comment (kaelemc) [inline docs/rn/0.56.md:41]: It's a switch not a router :)

File: docs/manual/kinds/vr-cat9kv.md [hunk 8]
```diff
@@ -20,12 +20,16 @@ The Q200 simulation has a limited featureset compared to the UADP simulation.
 
 ## Resource requirements
 
+The [[[ kind_display_name ]]] is a resource-hungry VM. When launched with the default settings, it requires the following resources:
```
Comment (kaelemc) [inline docs/manual/kinds/vr-cat9kv.md:23]: May be worth specifying by default is requires the UADPs minimum resources. The table just shows the recommended resources for each ASIC simulation. Q200 will use UADP's 4vCPU and 18432M RAM unless the user overwrites this.

**Review by kaelemc — COMMENTED** [review]
Looks good.

**Pull request merged**